Cargo.lock

```diff
@@ -78,9 +78,9 @@ checksum = "bef38d45163c2f1dde094a7dfd33ccf595c92905c8f8f4fdc18d06fb1037718a"
 
 [[package]]
 name = "bitflags"
-version = "2.0.2"
+version = "2.1.0"
```

Comment:
Please keep Cargo.lock changes to the minimum required to implement the feature. I'm guessing you called `cargo update` by mistake?

Reply:
Fixed